agot-bg-game-server/src/client/GameSettingsComponent.tsx

```diff
@@ -75,6 +75,24 @@ export default class GameSettingsComponent extends Component<GameSettingsCompone
                                 />
                             </Col>
                         </Row>
+                        <Row>
+                            <Col xs="auto">
+                                <FormCheck
+                                    id="random-houses-setting"
```

Comment:
Please change that to `westeros-phase-variant-setting`. Here the easiest way is fixing it locally and then `git commit --amend` followed by `git push --force`.

Reply:
Fixed

I remember changing that but somehow I think I copied from the wrong branch when I was trying to put everything on a single commit. It was 3 or 4 branches until I finally got it right.